pkg/asset/installconfig/gcp/validation.go

```diff
@@ -240,6 +240,11 @@ func ValidatePreExistingPublicDNS(client API, ic *types.InstallConfig) *field.Er
 		return nil
 	}
 
+	// skip validation when user configured dns is enabled
```

Comment:
Should this test be added [here](https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/platformprovisioncheck.go#L93) instead?